website/docs/connect-data/integrations.md

```diff
@@ -38,145 +38,150 @@ Browse through the tutorials below to see how to connect non-native datasources
+</div>
+
+<div class="containerGrid">
+    <div class="containerColumnSampleApp columnGrid column-one" style={{padding:"20px"}}>
         <div class="containerCol">
             <img class="containerImage" src="/img/shopify-logo_3YbQ78SX7.png" alt="Shopify-logo"/>
         </div> <hr/>
         <div class="containerDescription"><strong>Shopify</strong> is an e-commerce platform for online stores and retail point-of-sale systems.<br/><br/></div>
          <div class="containerTutorialLink"><a href="https://www.appsmith.com/blog/building-a-shopify-admin-panel-a-step-by-step-guide"><strong>View Tutorial</strong></a> </div>
     </div>
-</div>
-
-<div class="containerGrid">
-    <div class="containerColumnSampleApp columnGrid column-one" style={{padding:"20px"}}>
+    <div class="containerColumnSampleApp columnGrid column-two" style={{padding:"20px"}}>
         <div class="containerCol">
             <img class="containerImage" src="/img/Slack-logo.png" alt="Slack-logo"/>
         </div> <hr/>
         <div class="containerDescription"><strong>Slack</strong> is a messaging program designed specifically for the workplace.<br/><br/>  </div>
         <div class="containerTutorialLink"><a href="https://www.appsmith.com/blog/building-a-daily-standup-application-in-30-minutes">
         <strong >View Tutorial </strong></a></div>
     </div>
-    <div class="containerColumnSampleApp columnGrid column-two" style={{padding:"20px"}}>
+    <div class="containerColumnSampleApp columnGrid column-three" style={{padding:"20px"}}>
         <div class="containerCol">
             <img class="containerImage" src="/img/supabase-logo_JCWmwonfL.png" alt="Supabase-logo"/>
         </div> <hr/>
         <div class="containerDescription"><strong>Supabase</strong> is an open source Firebase alternative. <br/><br/> </div>
          <div class="containerTutorialLink"><a href="https://www.appsmith.com/blog/build-an-equipment-checkout-app-for-the-admin-team-at-your-company"><strong>View Tutorial</strong> </a>
          </div>
     </div>
-    <div class="containerColumnSampleApp columnGrid column-three" style={{padding:"20px"}}>
+</div>
+
+<div class="containerGrid">
+    <div class="containerColumnSampleApp columnGrid column-one" style={{padding:"20px"}}>
         <div class="containerCol">
             <img class="containerImage" src="/img/xano-logo_nnco8rx_b.png" alt="Xano-logo"/>
         </div> <hr/>
         <div class="containerDescription"><strong>Xano</strong> is the fastest no-code backend development platform.<br/><br/></div>
          <div class="containerTutorialLink"><a href="https://www.appsmith.com/blog/adding-social-authentication"><strong>View Tutorial</strong></a> </div>
     </div>
-</div>
-
-<div class="containerGrid">
-    <div class="containerColumnSampleApp columnGrid column-one" style={{padding:"20px"}}>
+    <div class="columnGrid column-two" style={{padding:"20px"}}>
```

Comment:
The formatting is broken for Zapier.
<img width="622" alt="Screenshot 2023-07-19 at 12 12 06 PM" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/11a232ee-4d6b-4a08-a56d-8cce4e890ad9">